src/boot/antelope.ts

```diff
@@ -43,17 +45,28 @@ export default boot(({ app }) => {
 
     // we need to wait 1000 milisec to ensure app.config.globalProperties?.$router is not null
     ant.events.onLoggedIn.subscribe({
-        next: async () => {
-            if (window.location.pathname === '/') {
-                (await getRouter(app)).push({ path: '/evm/wallet?tab=balance' });
+        next: async (account: AccountModel) => {
+            if (account.isNative) {
+                if (!window.location.pathname.startsWith('/zero')) {
+                    const router = await getRouter(app);
+                    router.push({ path: '/zero/balance' });
+                }
+            } else {
+                if (!window.location.pathname.startsWith('/evm')) {
+                    const router = await getRouter(app);
+                    router.push({ path: '/evm/wallet?tab=balance' });
+                }
```

Comment:
Now the redirection takes into account which type of chain you are logging in to. It only redirects you if you are in the wrong path.